### ISSUE TITLE
Entity Describer is slow with large number of entities

### ISSUE THREAD

**Issue by xfkoko** [drodil/backstage-plugin-toolbox#136]
Toolbox -> Entity Describer
Is slow to load all entities at the beginning. Also writing in the search box is "laggy" or not showing any loading indication.

Cause: toolbox fetches entities from /api/catalog/entities, when catalog has large amount of entities the initial search can take long time and lags

**Comment by drodil** [comment]
Yep, aknowledged. Same problem as with the default catalog table.

**Comment by xfkoko** [comment]
Not as bad though. It does not make request to the database per stroke from what I observed.

**Comment by drodil** [comment]
Should implement the selector with virtualized list.
